config/routes.rb

```diff
@@ -1,12 +1,15 @@
 Rails.application.routes.draw do
   root 'coffee_shops#index'
   devise_for :users
-  resources :coffee_shops, only: [:index, :show, :create, :new]
+
+  resources :coffee_shops, only: [:index, :create, :new]
+
+  get '/coffee_shops/:id', to: 'homes#index'
 
   namespace :api do
     namespace :v1 do
       resources :coffee_shops, only: [:show] do
-        resources :reviews, only: [:index]
+        resources :reviews, only: [:index, :new, :create]
```

Comment:
reviews is handled by an API endpoint, so we don't need a `new` router in an API controller. `new` would be to render a new Review form, but the form is displayed in React so this `new` API endpoint would never be hit. We also want to make sure these routes match what we have in our controllers - and your API Reviews controller only has a `create` route, so that is all that should be listed here (removing reviews index and new). An index of reviews would be used to show ALL reviews for ALL coffee shops, but when displaying reviews we'll probably want to actually go through a coffee shop show page to only show _its_ relevant reviews